packages/kit/src/core/generate_manifest/index.js

```diff
@@ -51,7 +61,7 @@ export function generate_manifest({ build_data, relative_path, routes, format =
 		_: {
 			entry: ${s(build_data.client.entry)},
 			nodes: [
-				${Array.from(bundled_nodes.values()).map(node => loader(node.path)).join(',\n\t\t\t\t')}
+				${Array.from(bundled_nodes.values()).filter(node => used_nodes.has(node.index)).map(node => loader(node.path)).join(',\n\t\t\t\t')}
```

Comment:
`bundled_nodes` is only used here, so instead of filtering afterwards, I would switch it around and do the `used_nodes` first and while constructing `bundled_nodes` use that. Also a little comment on the `if`-block why we do this, because from reading it might look duplicative ("isn't nodes and routes the same?")

Reply:
yeah, it's broken, am in the process of fixing